src/createAgentManager.ts

```diff
@@ -93,6 +107,34 @@ export function getAgent(agentId: string, auth: Auth, baseURL?: string): Promise
     return agentsApi.getById(agentId);
 }
 
+function getInitAnaliticsInfo(agent: Agent) {
+    const mobileOrDesktop= () => {
+        return /Mobi|Android/i.test(navigator.userAgent) ? 'Mobile' : 'Desktop'
+    };
+    const getUserOS = () => {
+        var platform = navigator.platform;
+        
+        if (platform.toLowerCase().includes("win")) {
+            return "Windows";
+        } else if (platform.toLowerCase().includes("mac")) {
+            return "MacOS";
+        } else if (platform.toLowerCase().includes("linux")) {
+            return "Linux";
+        } else {
+            return "Unknown"; // Unable to determine the OS
+        }
+    }
+    return {
+        os: `${getUserOS()} - ${mobileOrDesktop()}`,
```

Comment:
its mixing between agent data and logistic data as os
too coupled IMO

Reply:
changed